FWCore/Utilities/interface/TimeOfDay.h

```diff
@@ -2,13 +2,16 @@
 #define FWCore_Utilities_TimeOfDay_h
 
 #include <sys/time.h>
+#include <chrono>
 #include <iosfwd>
 
 namespace edm {
   struct TimeOfDay {
     TimeOfDay();
     explicit TimeOfDay(struct timeval const& tv);
 
+    explicit TimeOfDay(std::chrono::microseconds const& duration);
```

Comment:
I would imagine a generic constructor doing the `duration_cast` to `microseconds` to be useful as well, but I'm fine with starting with explicit `microseconds`.

Reply:
if you don't mind having it in the header file, I can make the change

Reply:
Having it in the header is fine, feel free to go ahead.